package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "data-capture-service",
-    "version": "6.2.6",
```

Comment:
As this is adding new functionality in a backwards compatible way, this should be a minor release rather than a patch.

If it was behind a toggle the patch would apply

Reply:
This is being merged into an RC, so I've removed all versioning for now. We can version the RC at deployment.